tests/e2e/core-tests/product_simple_01_explore_test.js

```diff
@@ -1,9 +1,10 @@
 const locator = require('../pages/locator');
 Feature('Simple product explore');
-Before(({ I }) => { // or Background
-    I.loginAsVendor();
-});
-Scenario('product edit explore test', ({ I }) => {
+// Before(({ I }) => { // or Background
+//     I.loginAsVendor();
+// });
+Scenario('product edit explore test', ({ I,loginAs }) => {
+    loginAs('vendor');
```

Comment:
Maintain upper and lower case for the user role because the user is case sensitive.